server/src/packages/landuse-form/landuse-form.service.ts

```diff
@@ -31,7 +31,7 @@ export class LanduseFormService {
       $filter=
         dcp_landuseid eq ${id}
       &$expand=
-        dcp_dcp_landuse_dcp_sitedatahform_landuseform($filter=statecode eq ${ACTIVE_STATECODE}),
+        dcp_dcp_landuse_dcp_sitedatahform_landuseform,
```

Comment:
Thanks! Can you make sure this is okay? Sometimes we get dinged by QA for allowing "inactive" things to appears.

Reply:
Oops, my bad. I only read the PR description not the actual bug description. Good catch!